### PR TITLE
Login functionality

### PR DESCRIPTION
**Pull request by michaelCasey**
- Retrieves user by email and password.
- User id is passed to next state through intent extras
- Made adjustment to student and tutor query to include the password

### CODE REVIEW

**Comment by ericdesj** [comment]
The eagle has landed :bird:

File: app/src/main/java/cs/dal/krush/models/Student.java [hunk 1]
```diff
@@ -68,8 +68,9 @@ public Cursor getDataByLastName(String lastName){
      * @param email
      * @return Cursor
      */
-    public Cursor getDataEmail(String email){
-        return dbRead.rawQuery("SELECT * FROM students WHERE email="+email+"",null);
+    public Cursor getDataEmail(String email, String password){
+        res = dbRead.rawQuery("SELECT * FROM students WHERE email=? AND password=?", new String[]{email, password});
```
Comment (Mons1Oerjan) [inline app/src/main/java/cs/dal/krush/models/Student.java]: no need to create `res`, just go `return dbRead.rawQuery(...)`

File: app/src/main/java/cs/dal/krush/models/Tutor.java [hunk 2]
```diff
@@ -109,8 +109,9 @@ public Cursor getDataByLastName(String lastName){
      * @param email
      * @return Cursor
      */
-    public Cursor getDataEmail(String email){
-        return dbRead.rawQuery("SELECT * FROM tutors WHERE email="+email+"",null);
+    public Cursor getDataEmail(String email, String password){
+        res = dbRead.rawQuery("SELECT * FROM tutors WHERE email=? AND password=?", new String[]{email, password});
```
Comment (Mons1Oerjan) [inline app/src/main/java/cs/dal/krush/models/Tutor.java]: same here

File: app/src/main/res/layout/student_home.xml [hunk 3]
```diff
@@ -47,5 +47,6 @@
         android:layout_height="270dp"
         android:layout_alignParentStart="true"
         android:layout_below="@+id/bookTutorLabel" />
+>>>>>>> fb622f5e25d6d3a64ca81045659423e5cffb0a21
```
Comment (Mons1Oerjan) [inline app/src/main/res/layout/student_home.xml]: remove this line

**Pull request merged**